release-notes/1.3.0.md

```diff
@@ -0,0 +1,4 @@
+# 1.3.0
+
+- Quickly restore visibility of messages read by /queue-admin/get-dlq-messages
+- TBC
```

Comment:
looks like there are other changes in 1.3.0 which need to be added too

Reply:
What are these other changes? If there is more stuff coming you should probably put a comment on the PR asking people not to publish this version yet...